frontend/src/js/controllers/challengeHostTeamsCtrl.js

```diff
@@ -198,7 +198,7 @@
             var confirm = $mdDialog.prompt()
                 .title('Invite others to this Team')
                 .textContent('Enter the email address of the person')
-                .placeholder('deshraj@cloudcv.org')
+                .placeholder('foobar@cloudcv.org')
```

Comment:
@Ram81 : please replace this text to "email" only. There is no need to add dummy email here.